### ISSUE TITLE
Groundtruth of timestep

### ISSUE THREAD

**Issue by longkhanh-fam** [ydqmkkx/ShallowFlowMatching-TTS#3]
May I ask about the value range of groundtruth of timestep in your dataset.

**Comment by ydqmkkx** [comment]
Hi, for flow matching models, we sample t from [0,1] during training.
Timestep is not a property of the data in the dataset.
So what is "groundtruth of timestep"?

**Comment by longkhanh-fam** [comment]
Thank you for your respense.
`t_loss = F.mse_loss(tp, th, reduction="mean")`
Regarding the th variable in the loss function, has its value ever reached $1.0$?

**Comment by ydqmkkx** [comment]
Oh thank you for your explanation.
You can read Table 3 and the tables in Appendix D (about α selection).
When α=1.0, the tg in the tables is the "groundtruth timestep".
In all of our experiments, this value converges to around 0.05~0.2

**Comment by longkhanh-fam** [comment]
Thank you for your instant response!
I now understand more about your interesting work. I may have further questions later, but for now I’ll close this issue.
